packages/wallets/src/features/cfd/modals/CTraderSuccessModal/CTraderSuccessModal.tsx

```diff
@@ -1,57 +1,40 @@
-import React, { useCallback } from 'react';
-import { useHistory } from 'react-router-dom';
+import React from 'react';
 import { useCtraderAccountsList } from '@deriv/api-v2';
-import { ModalStepWrapper, ModalWrapper, WalletButton, WalletButtonGroup } from '../../../../components';
+import { ModalStepWrapper, ModalWrapper } from '../../../../components';
 import { useModal } from '../../../../components/ModalProvider';
 import useDevice from '../../../../hooks/useDevice';
 import { THooks } from '../../../../types';
 import { PlatformDetails } from '../../constants';
 import { CFDSuccess } from '../../screens';
+import { CTraderSuccessModalButtons } from './components';
 
 type TCTraderSuccessModal = {
+    displayBalance: string;
```

Comment:
@lubega-deriv 
since we are also using `CTraderSuccessModal` in packages/wallets/src/features/cfd/screens/CompareAccounts/CompareAccountsButton.tsx, but it refers to real account, ket make this props as optional, because in packages/wallets/src/features/cfd/screens/CompareAccounts/CompareAccountsButton.tsx it can cause TS error

Reply:
actually we can see this error at the end of the file :)
![Uploading Screenshot 2024-03-29 at 11.33.18.png…]()

Reply:
Updated